test/mri/ruby/test_array.rb

```diff
@@ -2574,10 +2574,8 @@ def test_rindex2
     assert_raise(StopIteration) { e.next }
 
     o = Object.new
-    class << o; self; end.class_eval do
```

Comment:
Did you modify this or are we out of date on this file?  We normally do not change the test files we receive.